queries.sql

```diff
@@ -58,4 +57,15 @@ FROM animals
 GROUP BY species;
 SELECT species, AVG(escape_attempts)
 FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND  '2000-12-31'
-GROUP BY species;
\ No newline at end of file
+GROUP BY species;
+COMMIT;
+
+
+SELECT * FROM animals LEFT JOIN owners ON owners.id = owners_id WHERE owners_id= 4;
```

Comment:
- [ ] Please use `'Sam Smith'` here instead of using the `id` here. The point is that in a real scenario where you need to query some data, you'll not have this information and will only be able to search and query using this string. You can use subqueries to select the name of the owner here instead of using just the `id`.

Reply:
Hi @ShadyShawkat , i resolve this issue by using the following :
<img width="892" alt="1x" src="https://github.com/TamaraDimov/Vet_clinic_database/assets/116429687/98728b0a-c0d9-4e2a-9201-ee32211afb2f">